contrib/intel/jenkins/summary.py

```diff
@@ -943,8 +943,15 @@ def summarize_items(summary_item, logger, log_dir, mode):
 
 
     full_file_name = f'{log_dir}/{output_name}'
+    if (os.exists(full_file_name)):
+        open_opts = 'a'
+    else:
+        open_opts = 'w'
```

Comment:
`open_opts` doesn't seem to be used.

Reply:
Thank you! I wrote out the inline thing before putting it into the with statement and forgot to remove it.